### PR TITLE
cluster-glue: compile as 64-bit

### PR DESCRIPTION
**Pull request by grueni**

### CODE REVIEW

File: components/cluster/cluster-glue/Makefile [hunk 1]
```diff
@@ -52,11 +52,11 @@ COMPONENT_PREP_ACTION=(cd $(@D); $(ENV) $(CONFIGURE_ENV) $(CONFIG_SHELL) autogen
 
 COMPONENT_PRE_CONFIGURE_ACTION=($(CLONEY) $(SOURCE_DIR) $(@D))
```
Comment (AndWac) [inline components/cluster/cluster-glue/Makefile:55]: You need to set COMPONENT_REVISION

**Comment by xen0l** [comment]
Please, set CONFIGURE_BINDIR.64 like [here](https://github.com/OpenIndiana/oi-userland/blob/b6ca80a20fb2f5e4e777be54f0bdf34372fde345/components/shell/dash/Makefile#L41) and don't forget to bump COMPONENT_REVISION if COMPONENT_VERSION does not change.

**Review by xen0l — CHANGES_REQUESTED** [review]
Please, set CONFIGURE_BINDIR.64 like [here](https://github.com/OpenIndiana/oi-userland/blob/b6ca80a20fb2f5e4e777be54f0bdf34372fde345/components/shell/dash/Makefile#L41) and don't forget to bump COMPONENT_REVISION if COMPONENT_VERSION does not change.

**Comment by pyhalov** [comment]
Superseeded by https://github.com/OpenIndiana/oi-userland/pull/3560

**Pull request closed without merging**